### PR TITLE
A11y: Fix accessibility of 'Set color' button in Value Mappings modal

### PR DESCRIPTION
**Pull request by JoaoSilvaGrafana**
**What this PR does / why we need it**:

`LinkButton` with `onClick` is not accessible with the keyboard, so replaced it with `Button`

### CODE REVIEW

**Review by yaelleC — COMMENTED** [review]
Nice 👍 , could you add a couple of screenshots to the PR description to check the before and after?
Thanks

**Review by yaelleC — APPROVED** [review]
Actually, I am just realising it was a LinkButton before so visuals will be the same 🤦‍♀️ 

You have my ✅  (but let's wait for a codeowner's approval too 🙂 )

**Review by Elfo404 — APPROVED** [review]
LGTM 🚀

**Review by axelavargas — APPROVED** [review]
Wohoo! 🥳 great job @JoaoSilvaGrafana,

ps, all the labels are correct, feel free to merge whenever you want :)

**Pull request merged**
